logbook-servlet/src/test/java/org/zalando/logbook/servlet/MultiFilterTest.java

```diff
@@ -130,8 +132,8 @@ private LocalResponse getResponse(final Filter filter) throws IOException, Servl
         return captor.getValue();
     }
 
-    private LocalResponse getResponse(final ExampleController controller) throws IOException {
-        final ArgumentCaptor<LocalResponse> captor = ArgumentCaptor.forClass(LocalResponse.class);
+    private HttpServletResponse getResponse(final ExampleController controller) throws IOException {
+        final ArgumentCaptor<HttpServletResponse> captor = ArgumentCaptor.forClass(HttpServletResponse.class);
```

Comment:
It's probably easier this way, you won't need a cast above:

```suggestion
    private ServletResponseWrapper getResponse(final ExampleController controller) throws IOException {
        final ArgumentCaptor<ServletResponseWrapper> captor = ArgumentCaptor.forClass(ServletResponseWrapper.class);
```

Reply:
compiler doesn't let me for some reason, as `readBytes()` method expects `HttpServletResponse` type, and `ServletResponseWrapper` even thought it's decendent, doesn't work 
![image](https://github.com/user-attachments/assets/0240e331-a362-4ac4-9edf-22bbcdd4830d)

Reply:
Sorry, I updated my comment above. It keeps the `getResponse` method signature (method returns `LocalResponse`), so no cast or any other changes would be required in the test method.

![image](https://github.com/user-attachments/assets/e77889f1-3de1-4230-990e-976e28d1e937)

Reply:
ah, good point, updated the PR as per your suggestion